client/src/Movies/MovieCard.js

```diff
@@ -1,7 +1,25 @@
 import React from 'react';
 
-const MovieCard = props => {
-  return;
+const MovieCard = ({movie}) => {
+  const { title, director, metascore, stars } = movie;
+  return (
+    <div className="movie-card">
+      <h2>{title}</h2>
+      <div className="movie-director">
+        Director: <em>{director}</em>
+      </div>
+      <div className="movie-metascore">
+        Metascore: <strong>{metascore}</strong>
+      </div>
+      <h3>Actors</h3>
+      
+      {stars.map(star => (
+        <div key={star} className="movie-star">
```

Comment:
good job remembering to pass the key, make sure you understand why react wants you to pass a key as well.